src/ex4/server/clients/pokemon_client.js

```diff
@@ -1 +1,20 @@
-// The Pokemon Client (using axios) goes here
\ No newline at end of file
+// The Pokemon Client (using axios) goes here
+const dotenv = require("dotenv");
+const axios = require("axios");
+const pokemonApi = require("../../envModule.js").pokemonApi;
+dotenv.config();
+const envFile = process.env;
+class PokemonClient {
+	constructor() {}
+	async getPokemon(pokeID) {
+		try {
+			const pokemonObj = await (await axios.get(`${pokemonApi}${pokeID}`)).data;
```

Comment:
It's a bad practice to concatenate multiple different actions in one raw.
should separate the first fetch from your second action of extracting only specific property from it `.data` 
Also, I think we talked about it.

the `catch` here is misleading because you act as if the catch is your request error handling even though it's just a side effect of the above code crashing due to your trigger of `.data` without checking response status or if res.ok..